### PR TITLE
Rewrite of first page

### PR DESCRIPTION
**Pull request by ddgenome**
Rewrite landing page per discussions.  Remove old, unused theme
files.  Clean up extra.css

Towards #86

Please review the new text in docs/index.md.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,4 +1,128 @@
-<img style="float: left; padding-top: 7px; padding-right: 10px; padding-bottom: 20px;" src="images/atomist-logo-source.jpg">
+<img style="float:left; padding-top:7px; padding-right:10px; padding-bottom:15px;" src="images/atomist-logo.png">
+
+# Atomist Overview
+
+Atomist enables you to ship software in less time and respond to
+operational events more quickly.  By automating tasks, integrating
+tools, and putting timely, correlated, and actionable information in
+front of your team, Atomist makes software development and operational
```
Comment (jrday) [inline docs/index.md]: "in front of your team" and the team phrasing in this section feels like we could simplify it. e.g. "By automating tasks, integrating tools, and ~putting~ *providing* timely, correlated, and actionable information ~in front of your team~, Atomist makes software ~development and operational response~ *delivery* faster, more standardized, and more fun.

File: docs/index.md [hunk 2]
```diff
@@ -1,4 +1,128 @@
...
+response faster, more standardized, and more fun.
+
+Timely, correlated, and actionable information results from capturing
+your processes and operations in code.  Atomist knows the platforms
+and tools you already use: from GitHub to BitBucket, from Trello to
```
Comment (jrday) [inline docs/index.md]: I think this should list the tools currently supported and we update over time

File: docs/index.md [hunk 3]
```diff
@@ -1,4 +1,128 @@
...
+Jira, from Travis CI to Jenkins, from Kubernetes to Spinnaker.
+Atomist understands your code: Java, C#, JavaScript, Scala, Python,
+Clojure, even Dockerfiles and POMs.  Most importantly, Atomist
+understands the relationship between your code, your tools, your
+environments, and your running services and brings this information to
+your team members where they live: chat.
```
Comment (jrday) [inline docs/index.md]: "... brings this information to ~your team members~ *you* where you live: chat."

File: docs/index.md [hunk 4]
```diff
@@ -1,4 +1,128 @@
...
+
+If your team spends time upgrading dependencies across multiple
```
Comment (jrday) [inline docs/index.md]: If ~your team spends~ *you spend* time..."

File: docs/index.md [hunk 5]
```diff
@@ -1,4 +1,128 @@
...
+repositories, integrating code repositories and CI, configuring and
+managing deployment platforms, figuring out who to ask about an error
+in a log, or switching from IDE to chat to web browser ad infinitum,
+let Atomist remove the drudgery and reduce the context switching so
+your team can focus on what matters: shipping great software.
+
+## Atomist Is SaaS
```
Comment (jrday) [inline docs/index.md]: maybe "Atomist *is* SaaS" with lower-case "is". Somehow the capital "I" makes my eyes want to read ISaaS or IaaS

File: docs/index.md [hunk 6]
```diff
@@ -1,4 +1,128 @@
...
+
+The Atomist managed service consumes events from your systems: source
+code repositories, continuous integration servers, deployment
+platforms, and chat.  The information from these systems is correlated
+together: pushes with builds, artifacts with deployments, stack traces
+with commits.  The correlation of these events is then used to
+automatically respond: retry a build, start an integration test,
+message a committer.
+
+***The Atomist managed service provides:***
```
Comment (jrday) [inline docs/index.md]: "managed service" has a different meaning than SaaS. I think we mean "The Atomist service provides:"

File: docs/index.md [hunk 7]
```diff
@@ -1,4 +1,128 @@
...
+-   Detailed [Reference Documentation][refdoc]
+
+[getting-started]: getting-started/index.md
+[quick]: quick-starts/index.md
+[refdoc]: reference-docs/index.md
+
+You can also stay up to date with Atomist by following our
+blog, [The Composition][composition].
+
+[composition]: https://medium.com/the-composition
```
Comment (jrday) [inline docs/index.md]: https://the-composition.com/

**Comment by ddgenome** [comment]
These PR builds keep failing, presumably because the push build and the PR build are both trying to push to the gh-pages branch at about the same time. I should disable gh-pages for PR builds.

**Pull request merged**